docs/sources/alerting/unified-alerting/opt-in.md

```diff
@@ -6,11 +6,11 @@ weight = 128
 
 # Enable Grafana 8 Alerts
```

Comment:
```suggestion
# Switch to legacy dashboard alerts
```

Reply:
Alternately, "Upgrading to Grafana 8 alerts"

Reply:
I prefer `Upgrading to Grafana 8 alerts`